hooks.go

```diff
@@ -0,0 +1,61 @@
+package logging
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+
+	"github.com/bshuster-repo/logrus-logstash-hook"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/gemnasium/logrus-graylog-hook.v2"
+)
+
+func (c LogConfig) initLogstashHook(h LogHook) error {
+	if err := c.validateRequiredHookSettings(h, []string{"network"}); err != nil {
+		return err
+	}
+	network, _ := h.Settings["network"]
```

Comment:
I don't have the full picture of the lib, but I'd catch for `ok` here, but if ya think it isn't needed that's fine